neofetch

```diff
@@ -93,6 +93,9 @@ get_distro() {
                     "tiny") distro="${distro//version*}" ;;
                 esac
 
+            elif type -p tazpkg >/dev/null; then
+                distro="SliTaz $(< /etc/slitaz-release)"
```

Comment:
If I remove those two lines it just reports `OS: 5.0 x86_64` on the first line of output.